pg_stream_copy/protocol.py

```diff
@@ -56,6 +58,70 @@ def build_double_precision(value: float) -> bytes:
     return _build_value(pack('>d', value))
 
 
+def build_numeric(value: Decimal) -> bytes:
+    value_tuple = value.as_tuple()
```

Comment:
is it possible to describe here what this function actually do? What is the desired output?

Reply:
builtin Decimal method: `https://docs.python.org/2/library/decimal.html#decimal.Decimal.as_tuple`

Reply:
no, I mean `build_numeric` function :)

Reply:
Link to PG documentation describing the actual format is here: https://github.com/rtbhouse-apps/pg-stream-copy/pull/4/files/315a47c3d88db4b7e208429a61e3e30a774d91f4#diff-1c73ce27aaf2c678be333bb8ea7fa968R114